src/FixedDataTable.js

```diff
@@ -1056,6 +1056,21 @@ var FixedDataTable = React.createClass({
 
     this._scrollHelper.setViewportHeight(bodyHeight);
 
+    // This calculation is synonymous to Element.scrollTop
+    var scrollTop = firstRowIndex && Math.abs(firstRowOffset - this._scrollHelper.getRowPosition(firstRowIndex));
+    if (scrollTop >= 0) {
+      var maxScrollTop = scrollContentHeight - bodyHeight;
+      // Handle the case where the scrollTop is beyond the maxScrollTop, such as when the user is completely scrolled
+      // down and resizes the viewport to be smaller vertically
+      if (scrollTop > maxScrollTop || scrollTop === 0) {
```

Comment:
Unfortunately I couldn't come up with a way to do the scrollTop === 0 once. If the 0 logic is removed, scrolling slowly back up can cause the scrollTop to never update all the way back to 0. There may be a better way, but for now this seems to work well enough.